tests/test_sequential.py

```diff
@@ -0,0 +1,29 @@
+from drift.loop import backtest, train
+from drift.splitters import ExpandingWindowSplitter
+from drift.transformations.columns import OnlyPredictions
+from drift.utils.tests import generate_all_zeros
+
+from drift.transformations.sequential import SequentialTransformation
+import pandas as pd
+
+def test_sequential() -> None:
```

Comment:
We could chat about how we approach testing models on actual data, in a non-flaky way (so we get deterministic or highly likely results)!

Reply:
Yes! I think there was lot of progress on that on your side, so this will be definetly outdated. Let's talk on how I should change it before we merge.